cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -348,3 +381,25 @@ func payloadBytes(blobRef string) ([]byte, error) {
 	}
 	return blobBytes, nil
 }
+
+func payloadDigest(blobRef string) (string, []byte, error) {
+	hexAlg, hexDigest, ok := strings.Cut(blobRef, ":")
+	if !ok {
+		return "", nil, fmt.Errorf("invalid digest format")
+	}
+	digestBytes, err := hex.DecodeString(hexDigest)
+	if err != nil {
+		return "", nil, err
+	}
+	return hexAlg, digestBytes, nil
+}
+
+func loadTrustedRoot(ctx context.Context, trustedRootPath string) (*root.TrustedRoot, error) {
+	if trustedRootPath != "" {
+		return root.NewTrustedRootFromPath(trustedRootPath)
+	}
+	ui.Infof(ctx, "no --trusted-root specified; fetching public good instance verification material via TUF")
```

Comment:
I'd guess the vast majority of use cases won't specify a trusted root, so I'd remove this.